agent/src/agent/cli/destination.py

```diff
@@ -78,6 +78,7 @@ def _start_monitoring_pipeline():
             pipeline_ = pipeline.manager.create_object('Monitoring', MONITORING_SOURCE_NAME)
             pipeline_manager = pipeline.manager.PipelineManager(pipeline_)
             click.secho('Starting Monitoring pipeline...')
+            # todo
```

Comment:
todo what?)

Reply:
I'm going to change the next two lines that create directories for sources and pipelines, it shouldn't happen during destination creation